packages/documentation-site/patternfly-docs/content/design-guidelines/styles/colors-for-charts/colors-grid.js

```diff
@@ -1,409 +1,83 @@
-          <div className="ws-content-chart-colors-gallery--information">
-            <div>#8F4700</div>
-            <div><code className="ws-content-css-var-text ws-code">--pf-chart-color-orange-500</code></div>
-          </div>
-        </GridItem>
-      </Grid>
-    </GridItem>
-    <GridItem sm={12} md={6} lg={4}>
-      <h3 className="pf-v5-c-title pf-m-xl ws-heading ws-title ws-h3">Red family</h3>
-      <Grid>
-        <GridItem span={2}>
-          <div className="ws-content-chart-colors ws-content-chart-colors-gallery--red-1"></div>
-        </GridItem>
-        <GridItem span={10}>
-          <div className="ws-content-chart-colors-gallery--information">
-            <div>#C9190B</div>
-            <div><code className="ws-content-css-var-text ws-code">--pf-chart-color-red-100</code></div>
-          </div>
-        </GridItem>
-        <GridItem span={2}>
-          <div className="ws-content-chart-colors ws-content-chart-colors-gallery--red-2"></div>
-        </GridItem>
-        <GridItem span={10}>
-          <div className="ws-content-chart-colors-gallery--information">
-            <div>#A30000</div>
-            <div><code className="ws-content-css-var-text ws-code">--pf-chart-color-red-200</code></div>
-          </div>
-        </GridItem>
-        <GridItem span={2}>
-          <div className="ws-content-chart-colors ws-content-chart-colors-gallery--red-3"></div>
-        </GridItem>
-        <GridItem span={10}>
-          <div className="ws-content-chart-colors-gallery--information">
-            <div>#7D1007</div>
-            <div><code className="ws-content-css-var-text ws-code">--pf-chart-color-red-300</code></div>
-          </div>
-        </GridItem>
-        <GridItem span={2}>
-          <div className="ws-content-chart-colors ws-content-chart-colors-gallery--red-4"></div>
-        </GridItem>
-        <GridItem span={10}>
-          <div className="ws-content-chart-colors-gallery--information">
-            <div>#470000</div>
-            <div><code className="ws-content-css-var-text ws-code">--pf-chart-color-red-400</code></div>
-          </div>
-        </GridItem>
-        <GridItem span={2}>
-          <div className="ws-content-chart-colors ws-content-chart-colors-gallery--red-5"></div>
-        </GridItem>
-        <GridItem span={10}>
-          <div className="ws-content-chart-colors-gallery--information">
-            <div>#2C0000</div>
-            <div><code className="ws-content-css-var-text ws-code">--pf-chart-color-red-500</code></div>
-          </div>
-        </GridItem>
-      </Grid>
-    </GridItem>
+const colorFamilies = [
+  {headingText: 'Blue', varColor: 'blue'},
+  {headingText: 'Green', varColor: 'green'},
+  {headingText: 'Cyan', varColor: 'cyan'},
+  {headingText: 'Purple', varColor: 'purple'},
+  {headingText: 'Gold', varColor: 'gold'},
+  {headingText: 'Orange', varColor: 'orange'},
+  {headingText: 'Red', varColor: 'red'},
+  {headingText: 'Grey', varColor: 'black'}
```

Comment:
I noticed in the main design guidelines color page, we're using "gray", and using "grey" here. Though I pinged @mceledonia about using "black" here instead, since that's the color name in the code, and also what designers tend to give developers (for example, "black-300" and "green-300"), and he suggested updating the wording to reflect the actual color name - so maybe we can change this to "black"? cc @mceledonia @mmenestr can you confirm?